changelogs/unreleased/igrishnov/gh-7479-bug-fix-default-uri-tarantoolctl-connect.md

```diff
@@ -0,0 +1,9 @@
+## bugfix/tarantoolctl
+
+* Made host=localhost by default for tarantoolctl connect.
+  Now, when specifying only the port, the connection is made
+  to localhost. This bugfix required to supplement the
+  implementation of the `src/lib/uri` parser.
+  Before this fix a call `uri.parse(uri.format(uri.parse(3301)))`
+  returned an error of 'Incorrect URI'.
+  Now this call return correct `service: '3301'` (gh-7479).
```

Comment:
Nit: No need to go deep into details here - it's a changelog, not a documentation or ticket, it's supposed to be brief. You can leave it as is though - up to you.

Reply:
Done, simplified